index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.3.0/css/all.min.css"
+      integrity="sha512-SzlrxWUlpfuzQ+pcUCosxcglQRNAq/DZjVsC0lE40xsADsfeQoEypE+enwcOiGjk/bSuGGKHEyjSoQ1zVisanQ=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <title>Wasyihun</title>
+  </head>
+  <body>
+    <header class="navbar">
+      <div>
+        <a class="Was_logo">Wasyihun</a>
+      </div>
+      <nav class="menu_icon">
+        <div class="menu_icon"></div>
+        <div class="menu_icon"></div>
+      </nav>
```

Comment:
Please consider making the hamburger icon just like as used in the template. There is an indifferent spaces between the lines of the icon.

## Remember:-

Rather than creating this icon using CSS styling, you can just import it from the design as you imported other images.
Or you can work on the styles to make it same

## Your Design:-

![image](https://user-images.githubusercontent.com/94223367/220410462-ab8b408c-b039-4310-9851-3c703157961a.png)

Reply:
@ABDULALI3468
 Ohh come on, I design exactly the same as the figma template  and  using the same style as figma may not display the out put b.c  it does not much with the screen...  i see in the loom video that they said "don't use exactly this style"

Can we have 5 min zoom meeting for tomorrow. Please  adjust your time.


I think those comments are not included in the code review criteria.